packages/core/aqua-to-js-compiler/src/utils.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { NonArrowType, ProductType } from '@fluencelabs/interfaces';
+import { readFile } from 'fs/promises';
+import path from 'path';
+
+interface PackageJson {
+    version: string;
+    dependencies: {
+        ['@fluencelabs/aqua-api']: string
+    }
+}
+
+export async function getPackageJsonContent(): Promise<PackageJson> {
+    const content = await readFile(path.join('..', 'package.json'));
```

Comment:
this shouldn't work still. Are you readiing file relative to what? This will not read file relative to this file location. I think you have to use import.meta.url or something

Reply:
Yeap, fixed